datacats/cli/manage.py

```diff
@@ -66,15 +67,16 @@ def reload_(environment, opts):
     """Reload environment source and configuration
 
 Usage:
-  datacats reload [-bp] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT [PORT]]
+  datacats reload [-b] [-p|--no-reload] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT [PORT]]
```

Comment:
I try my best to provide you with the best philosophical insights in my code. :stuck_out_tongue: 

But yeah this naming sucks. Any ideas?